proguard-com.twitter.sdk.android.twitter.txt

```diff
@@ -2,6 +2,9 @@
 # '-include' this file in your proguard config
 # Autogenerated file -- Do not modify
 
+#Proguard Config for when AppCompat is not a dependency
+-dontwarn  com.digits.sdk.android.*ActionBarActivity
+
```

Comment:
In order to support AppCompat >21 and Material Theme your activities are force to extend from ActionBarActivity